pydoro/pydoro_core/tomato.py

```diff
@@ -262,6 +263,8 @@ def __init__(self, tomato):
         self._task = Tasks.WORK
         self._status = TaskStatus.STARTED
         self._started_at = cur_time()
+        if len(self._tomato.configs.work_state_cmd) > 0:
```

Comment:
len is not needed here `if self._tomato.configs.work_state_cmd:` would do the same thing. :)